src/promptflow/tests/executor/e2etests/test_traces.py

```diff
@@ -97,11 +97,11 @@ def validate_openai_apicall(self, apicall: dict):
         """
         get_trace = False
         if apicall.get("name", "") in (
-            "openai.api_resources.chat_completion.ChatCompletion.create",
-            "openai.api_resources.completion.Completion.create",
-            "openai.api_resources.embedding.Embedding.create",
-            "openai.resources.completions.Completions.create",  # openai>=1.0.0
-            "openai.resources.chat.completions.Completions.create",  # openai>=1.0.0
+            "openai_completion_legacy",
+            "openai_chat_legacy",
+            "openai_embedding_legacy",
+            "openai_chat",  # openai>=1.0.0
+            "openai_completion",  # openai>=1.0.0
```

Comment:
Do we need to add async names here?
![image](https://github.com/microsoft/promptflow/assets/17527303/49f1a074-5eba-469d-b4f3-444bd43ec5e0)

Reply:
Will add in next PR.